ModelViewer/ModelViewer/ModelViewer.vcxproj

```diff
@@ -59,6 +57,9 @@
       <Message>Copy dll files to output folder</Message>
       <Command>xcopy /s /y "$(SOLUTIONDIR)ThirdParty\assimp5.0.1\bin\Debug\*.dll" "$(OUTPUTPATH)"</Command>
     </PostBuildEvent>
+    <ClCompile>
+      <AdditionalIncludeDirectories>C:\workspace\ModelViewer\ModelViewer\ThirdParty\assimp5.0.1\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Does the $(SOLUTIONDIR) marco not work for you?  It's best if we can avoid static paths in these files.

Reply:
My bad. I'll swap out my settings. Every once in a while vs code ax's my settings.